packages/slider/src/Slider.ts

```diff
@@ -17,7 +17,7 @@ import {
     TemplateResult,
     query,
     PropertyValues,
-} from 'lit-element';
+} from '@spectrum-web-components/base';
```

Comment:
I noticed a ux bug with the `rtl` slider:

![slider_bug](https://user-images.githubusercontent.com/8894489/89222295-abeea100-d589-11ea-8905-330b33227c13.gif)

We may need to update our style getters to support the `rtl` direction

Reply:
I added some of the storybook settings to make testing this easier, but my plan is to add it systematically after all of the baseline PRs are in and then turn on visual regressions for those URLs as well.